app/assets/javascripts/code/services/_draw_session.js

```diff
@@ -104,7 +104,7 @@ export const drawSession = (
     },
 
     clear: function(sessions) {
-      sessions.forEach(this.undoDraw);
+      _(sessions).each(_(this.undoDraw).bind(this));
```

Comment:
Wow this line is really noisy. Well it seems there's a null somewhere. Since 
```
_(null) === []
```
this way it does not explode. Prolly
```
(sessions || []).forEach(this.undoDraw);
```
is enough? Would be good to investigate why there's a null passed in

Reply:
That's my assumption since the error is
```
angular.js:10147 TypeError: Cannot read property 'forEach' of undefined
    at o.clear (_draw_session.js:107)
    at o.redraw (_draw_session.js:96)
```
which I guess means that the following happened
```
undefined.forEach(this.undoDraw)
```